sklearn/datasets/_samples_generator.py

```diff
@@ -607,7 +610,7 @@ def make_circles(n_samples=100, shuffle=True, noise=None, random_state=None,
         See :term:`Glossary <random_state>`.
 
     factor : 0 < double < 1 (default=.8)
-        Scale factor between inner and outer circle.
+        Scale factor between radii of inner and outer circle.
```

Comment:
Hmm, I guess few people know this word

Reply:
Well, when one deals with circles, one usually knows what radius is ;)
I agree that plural form is less know. Anyway, I changed it because when I read description I wasn't sure what scale it is exactly. But this is not the scope of my changes, so I restored previous version.